srcpkgs/glibc/template

```diff
@@ -133,7 +128,7 @@ do_install() {
 	( cd build && make install_root=${DESTDIR} install )
 
 	# The C.UTF-8 locale should always be available
-	if [ -z "${CROSS_BUILD}" ]; then
+	if [ -z "${CROSS_BUILD}" ] || [ "${XBPS_MACHINE%-*}" = "${XBPS_TARGET_MACHINE%-*}" ]; then
```

Comment:
You could just drop the `[ -z "${CROSS_BUILD}" ]` test here.

Reply:
Alright.